### ISSUE TITLE
Getting `client.clientConfig is undefined` typeerror when trying to upload video

### ISSUE THREAD

**Issue by phmasek** [sanity-io/sanity-plugin-mux-input#213]
### Getting this error when trying to upload a file.
![image](https://user-images.githubusercontent.com/1712993/220424705-2000a3c8-878d-457e-b00b-ad1e6d09cfdf.png)

### Following the errors in the console this is what we get:
![image](https://user-images.githubusercontent.com/1712993/220425322-de74c968-54ae-4b2e-91aa-dc9f4dbeb8d8.png)

### `upload.ts:118`:
![image](https://user-images.githubusercontent.com/1712993/220425382-4207b0b8-cbab-4d2f-8ecd-a086057d0a13.png)

### `this.props.client`
![image](https://user-images.githubusercontent.com/1712993/220425535-e743b752-b55f-4134-bc71-38ebeab49608.png)

### `Input.tsx`
![image](https://user-images.githubusercontent.com/1712993/220425737-41021e07-ac52-4687-94b5-12bb1804acbd.png)

**Comment by guictrl** [comment]
Same here: 
<img width="639" alt="image" src="https://user-images.githubusercontent.com/47580888/220815916-857f8338-ed58-4941-af3f-894f7e68a90f.png">

**Comment by phmasek** [comment]
This error started when Sanity migrated to using @sanity/client v5. The migration happened at v3.4.0 so if you rollback Sanity to v3.3.1 you should be able to use sanity-plugin-mux-input normally.

**Comment by bobinska-dev** [comment]
Hello everyone, 
I see that you are setting up the client in the input like without a version. 
Please add it like [this](https://www.sanity.io/docs/migration-cheat-sheet#:~:text=const%20client%20%3D%20useClient(%7BapiVersion%3A%20%272021%2D10%2D21%27%7D)): 

```js
  const client = useClient({apiVersion: '2021-10-21'})
```

**Comment by guictrl** [comment]
Did anyone get this working? Changing to `sanity@3.3.1` got me a bit further along in the upload, though I'm getting this new error now:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/47580888/221051806-b18a54d0-a8d5-4e79-bceb-43de2f11bf49.png">

package.json dependencies for context:
```
"dependencies": {
    "@sanity/orderable-document-list": "^1.0.2",
    "@sanity/vision": "^3.0.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-is": "^18.2.0",
    "sanity": "^3.3.1",
    "sanity-plugin-mux-input": "2.0.11",
    "styled-components": "^5.2.0"
}
```

**Comment by roblom** [comment]
> Could you try out the just-released [v2.0.12](https://github.com/sanity-io/sanity-plugin-mux-input/releases/tag/v2.0.12) and see if it fixes it? It includes a related [bugfix](https://github.com/sanity-io/sanity-plugin-mux-input/pull/214) submitted by @phmasek 😄

This does not resolve the issue when sanity version is > 3.3.1, tried with 3.4.0 and 3.6.0

Meaning: our dependency on video content stops us from upgrading sanity :-\
We're stuck using sanity 3.3.1 + mux plugin 2.0.8
@stipsan , a working combination of latest sanity + mux with *signed* videos is much desired.

mux: 2.0.12 + sanity 3.4.0 brings back this
![image](https://user-images.githubusercontent.com/738522/227898732-b7ef4b57-11da-4f2f-8953-9d9aa0010c39.png)

**Comment by jonabc** [comment]
@roblom it looks like you've reported that problem [separately](https://github.com/sanity-io/sanity-plugin-mux-input/issues/239), which was fixed with [this change](https://github.com/sanity-io/sanity-plugin-mux-input/pull/253) included in the v2.0.17 release.

It looks like the bug reported in this issue has been resolved so I'm going to close this.  If you're encountering any other problems, please open a new issue to let us know 🙏
